package.json

```diff
@@ -87,7 +88,17 @@
     "gh-pages": "0.12.0",
     "html-webpack-plugin": "2.28.0",
     "imports-loader": "0.7.1",
+    "istanbul-instrumenter-loader": "^2.0.0",
```

Comment:
I also locked `istanbul-instrumenter-loader`'s version to `"2.0.0"` to make it more consistent with the rest of the packages.